decidim-core/lib/decidim/core.rb

```diff
@@ -587,6 +587,37 @@ def self.reset_all_column_information
     {}
   end
 
+  def self.open_data_manifests
```

Comment:
I always forgot about this, but we shouldn't be adding new `OpenStruct` calls. See https://www.rubydoc.info/gems/rubocop/0.61.1/RuboCop/Cop/Performance/OpenStruct

An alternative modern way of doing this is using the Data class: https://docs.ruby-lang.org/en/3.2/Data.html 

A quick proof of concept: 

```ruby
Manifests = Data.define(:name, :collection, :serializer, :include_in_open_data)
Manifests.new(
  name: :users,
  collection: ->(organization) { Decidim::User.where(organization:).confirmed.not_blocked.includes(avatar_attachment: :blob) },
  serializer: Decidim::Exporters::OpenDataUserSerializer,
  include_in_open_data: true
)
  ```

Reply:
done in [d98aedf](https://github.com/decidim/decidim/pull/13568/commits/d98aedfa22ecd1c3874a0f60b6c71b735725efb7)